tools/distrib/check_boringssl_commit.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
nit:  check_boringssl_commit.sh is not a great name.  You are checking if the "shadow boringssl symbol list is up to date", the fact that you are looking whether the commit sha in a file is the same is implementation detail. IMHO, seeing "commit" in the script name doesn't tell me anything useful

I suggest renaming to `check_boringssl_symbol_list.sh`, `check_boringssl_shadow_symbol_list.sh` or sth in that sense.

Reply:
`check_shadow_boringssl_symbol_list.sh` is perhaps the best name, but leaving up to you.